draft-birkholz-scitt-software-use-cases.md

```diff
@@ -211,45 +211,38 @@ There is no standardized way to:
-
-* How does the client applying the firmware update on the system know that the received firmware is not faulty or malicious?
-
-* What if the signing identity used to assert the authenticity of the firmware is somehow used to sign unintended updates?
-
-* How can one ascertain that the released firmware is not subverted or compromised due to an insider risk - be it malicious or otherwise?
-
-* How does the publisher even know that their deliverable has been compromised? Can they trust their key protection or audit logging?
+There is no standardized way to:
 
-* How does the update client on an instance of a health monitoring system know that they have been given the same update as all other devices or one specially crafted for just a small subset of a fleet of devices?
+* provide an update framework which allows verification of integrity and authenticity of firmware updates before installation
+* reliabily discern an update that has been signed by the appropriate and intended signing identity
+* clearly communicate firmware revisioning and security checks completed to ensure the firmware is safe for install
 
 ## Software Integrator assembling a software product for a smart car
 
-Software Integration is a complex activity. This typically involves getting various software components from multiple suppliers and producing an integrated package deployed as part of device assembly.
-
-Car manufacturers source integrated software for their autonomous vehicles from third parties that integrates software components from various sources. Integration complexity creates a higher risk of security vulnerabilities to the delivered software.
-
-### Software Problem Summary
+Software Integration is a complex activity. This typically involves getting various software components from multiple suppliers and producing an integrated package deployed as part of device assembly. For example, car manufacturers source integrated software for their autonomous vehicles from third parties that integrates software components from various sources. Integration complexity creates a higher risk of security vulnerabilities to the delivered software.
 
-* While the software runs on the automated vehicle, periodic vulnerability scanning software detects a known security issue with one component. End User gets a "Warning Indicator" on the dashboard. As a result it reports the problem to the car manufacturer. It is then subsequently notified to the integrator.
+Consumer of an integrated software wants:
 
-* Integrator analysis leads to a suspected issue with the supplied Operating System (OS) software from an Independent Software Vendor (ISV). It demands specific environment and architectural details associated with the built OS binary to ascertain that the software was produced without tampering by Vendor
+* Bill of material capturing all software components used, with the SBOM stored in a secure tamperproof location
```

Comment:
Suggest:
* to receive an alert when a vulnerability scan detects a known security issue on a running software component.